docs/releasing/README.md

```diff
@@ -170,11 +170,11 @@ Once the tag is created, you need to test it on the Fuji testnet both locally an
     git clone git@github.com:ava-labs/avalanchego.git
     ```
 
-1. Build AvalancheGo using those VM ids:
+1. Checkout correct AvalancheGo version, the version should match the one used in Subnet-EVM `go.mod` file
 
     ```bash
     cd avalanchego
-    ./scripts/build.sh
+    git checkout v1.13.0
```

Comment:
Seems like they should just check out the branch you need with:
```
git clone git@github.com:ava-labs/avalanchego.git --branch v1.13.0
```

Don't they still need to run `./scripts/build.sh`? Even if it's only to verify that it builds fine?